aws/extensions/computetask_awswin_cwlog/extension.ftl

```diff
@@ -0,0 +1,245 @@
+                                r'                            "ERROR", ',
+                                r'                            "CRITICAL" ',
+                                r'                        ], ',
+                                r'                        "event_name": "System", ',
+                                r'                        "log_group_name": "' + logGroupName + '", ',
+                                r'                        "log_stream_name": "{instance_id}/system" ',
+                                r'                    }, ',
+                                r'                    { ',
+                                r'                        "event_format": "xml", ',
+                                r'                        "event_levels": [ ',
+                                r'                            "WARNING", ',
+                                r'                            "ERROR", ',
+                                r'                            "CRITICAL" ',
+                                r'                        ], ',
+                                r'                        "event_name": "Application", ',
+                                r'                        "log_group_name": "' + logGroupName + '", ',
+                                r'                        "log_stream_name": "{instance_id}/application" ',
+                                r'                    }, ',
+                                r'                    { ',
+                                r'                        "event_format": "xml", ',
+                                r'                        "event_levels": [ ',
+                                r'                            "VERBOSE", ',
+                                r'                            "INFORMATION", ',
+                                r'                            "WARNING", ',
+                                r'                            "ERROR", ',
+                                r'                            "CRITICAL" ',
+                                r'                        ], ',
+                                r'                        "event_name": "Security", ',
+                                r'                        "log_group_name": "' + logGroupName + '", ',
+                                r'                        "log_stream_name": "{instance_id}/security" ',
+                                r'                    } ',
+                                r'                ] ',
+                                r'            } ',
+                                r'        } ',
+                                r'    }, ',
+                                r'    "metrics": { ',
+                                r'        "append_dimensions": { ',
+                                r'            "AutoScalingGroupName": "${aws:AutoScalingGroupName}", ',
+                                r'            "ImageId": "${aws:ImageId}", ',
```

Comment:
Image Id might be  a bit far here, I think just ASG, InstanceId and Type are good dimensions

Reply:
Cool